src/runtime_src/tools/scripts/service_bundle.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2019-2021 Xilinx, Inc. All rights reserved.
```

Comment:
Hmmm....Is this a new file or was it moved.   If new, then the copyright date needs to be corrected.

Reply:
fixed.